src/main/resources/ArchitectureReloadedBundle.properties

```diff
@@ -24,4 +24,6 @@ analyzing=Analyzing
 color.action.text=Highlighting
 color.action.description=Highlight refactorings in table
 other.settings=Other settings
-search.for.move.field.refactorings=Search for "Move field" refactorings
\ No newline at end of file
+search.for.move.field.refactorings=Search for "Move field" refactorings
+empty.scope.dialog.title = Empty Scope
+empty.scope.dialog.message = There are no files in selected scope
```

Comment:
I believe we should add some recommendations here suggesting users what to do. For example, open Project Structure -> Modules dialog and mark some directories as Sources. It would be great if this "Project Structure -> Modules" label will be a link that opens the dialog.